raiden/src/store/actions.ts

```diff
@@ -1,526 +1,178 @@
-  token: string;
-  tokenNetwork: string;
-  first: boolean; // first time monitoring this token, i.e. just started monitoring
-}
-
-export interface ChannelOpenAction extends RaidenAction {
-  type: RaidenActionType.CHANNEL_OPEN;
-  tokenNetwork: string;
-  partner: string;
-  settleTimeout: number;
-}
-
-export interface ChannelOpenedAction extends RaidenAction {
-  type: RaidenActionType.CHANNEL_OPENED;
-  tokenNetwork: string;
-  partner: string;
-  id: number;
-  settleTimeout: number;
-  openBlock: number;
-  txHash: string;
-}
-
-export interface ChannelOpenActionFailed extends RaidenActionFailed {
-  type: RaidenActionType.CHANNEL_OPEN_FAILED;
-  tokenNetwork: string;
-  partner: string;
-}
-
-export interface ChannelMonitoredAction extends RaidenAction {
-  type: RaidenActionType.CHANNEL_MONITORED;
-  tokenNetwork: string;
-  partner: string;
-  id: number;
-  fromBlock?: number;
-}
-
-export interface ChannelDepositAction extends RaidenAction {
-  type: RaidenActionType.CHANNEL_DEPOSIT;
-  tokenNetwork: string;
-  partner: string;
-  deposit: BigNumber;
-}
-
-export interface ChannelDepositedAction extends RaidenAction {
-  type: RaidenActionType.CHANNEL_DEPOSITED;
-  tokenNetwork: string;
-  partner: string;
-  id: number;
-  participant: string;
-  totalDeposit: BigNumber;
-  txHash: string;
-}
-
-export interface ChannelDepositActionFailed extends RaidenActionFailed {
-  type: RaidenActionType.CHANNEL_DEPOSIT_FAILED;
-  tokenNetwork: string;
-  partner: string;
-}
-
-export interface ChannelCloseAction extends RaidenAction {
-  type: RaidenActionType.CHANNEL_CLOSE;
-  tokenNetwork: string;
-  partner: string;
-}
-
-export interface ChannelClosedAction extends RaidenAction {
-  type: RaidenActionType.CHANNEL_CLOSED;
-  tokenNetwork: string;
-  partner: string;
-  id: number;
-  participant: string;
-  closeBlock: number;
-  txHash: string;
-}
-
-export interface ChannelCloseActionFailed extends RaidenActionFailed {
-  type: RaidenActionType.CHANNEL_CLOSE_FAILED;
-  tokenNetwork: string;
-  partner: string;
-}
-
-export interface ChannelSettleableAction extends RaidenAction {
-  type: RaidenActionType.CHANNEL_SETTLEABLE;
-  tokenNetwork: string;
-  partner: string;
-  settleableBlock: number;
-}
-
-export interface ChannelSettleAction extends RaidenAction {
-  type: RaidenActionType.CHANNEL_SETTLE;
-  tokenNetwork: string;
-  partner: string;
-}
-
-export interface ChannelSettledAction extends RaidenAction {
-  type: RaidenActionType.CHANNEL_SETTLED;
-  tokenNetwork: string;
-  partner: string;
-  id: number;
-  settleBlock: number;
-  txHash: string;
-}
-
-export interface ChannelSettleActionFailed extends RaidenActionFailed {
-  type: RaidenActionType.CHANNEL_SETTLE_FAILED;
-  tokenNetwork: string;
-  partner: string;
-}
-
-export interface MatrixSetupAction extends RaidenAction {
-  type: RaidenActionType.MATRIX_SETUP;
+}>();
+
+export const newBlock = createStandardAction('newBlock')<{ blockNumber: number }>();
```

Comment:
Single-property action payloads could be the payload-type itself, but for always having an object there is good for consistency as well as using the property name also as a self-documenting constraint